src/mbed_tools/build/build.py

```diff
@@ -38,6 +42,7 @@ def generate_build_system(
 
 def _cmake_wrapper(*cmake_args: str) -> None:
     try:
+        logger.debug("Running CMake with args: %s", cmake_args)
```

Comment:
Is this for verbose mode? How do users see this?

Reply:
Yes, it's for verbose mode. `-vvv` enables debug level logging.